src/mars.d

```diff
@@ -136,6 +136,7 @@ Where:
   -Hf=<filename>   write 'header' file to filename
   --help           print help and exit
   -I=<directory>   look for imports also in directory
```

Comment:
If you look at -HF, it accepts an optional leading =. This was done in the last year to match unix-y conventions. And, in fact, -I already accepted an optional leading =.